flow_screen_components/richTextInputFSC/README.md

```diff
@@ -1,3 +1,45 @@
 # Rich Text Edit Control #
 
 https://unofficialsf.com/rich-text-input-flow-screen-component/
+
+
+# inputRichTextFSC
+Rich Text enhancements for Flows
+
+### Summary:
+IMPORTANT:  Use richerText = true input attribute if you want to leverage these enhanced features.  Otherwise leave empty and regular input Rich Text component will be used.
+1)	Block disallowed words and symbols
+    a.	Control whether this is a block that prevents next/finish of flow.
```

Comment:
Perhaps phrase this "Warn or Block Transition if disallowed words and/or symbols are present"

Reply:
Great, thanks.  This will be in next update.